### PR TITLE
Improve evaluation methodology

### PR DESCRIPTION
**Pull request by daemon**
- Evaluate using the inference engine on examples containing entire "Hey Firefox"
- Remove frame-level evaluation
- Use three keywords and 500ms window with res8 for best results

### CODE REVIEW

File: ww4ff/align/stub.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from .base import AlignedTranscription
+
+
+__all__ = ['StubAligner']
+
+
+class StubAligner:
```
Comment (ljj7975) [inline ww4ff/align/stub.py:9]: just out of curiosity, where does the name StubAligner come from?

File: ww4ff/data/transform/base.py [hunk 2]
```diff
@@ -134,7 +136,10 @@ def __call__(self, examples: Sequence[WakeWordClipExample]) -> ClassificationBat
                 new_examples.append((self.negative_label, ex.emplaced_audio_data(ex.audio_data[..., a:b])))
         new_examples = sorted(new_examples, key=lambda x: x[1].audio_data.size()[-1], reverse=True)
         lengths = torch.tensor([ex.audio_data.size(-1) for _, ex in new_examples])
-        max_length = max(ex.audio_data.size(-1) for _, ex in new_examples)
+        if self.pad_to_window:
+            max_length = int(self.sample_rate * self.window_size_ms / 1000)
+        else:
+            max_length = max(ex.audio_data.size(-1) for _, ex in new_examples)
```
Comment (ljj7975) [inline ww4ff/data/transform/base.py:142]: what if the audio size > max_length?

File: ww4ff/align/stub.py (hunk 1 above)
Comment (daemon) [reply ww4ff/align/stub.py:9]: It's a class stub that just produces some mock alignment.

File: ww4ff/data/transform/base.py (hunk 2 above)
Comment (daemon) [reply ww4ff/data/transform/base.py:142]: That should never happen, since the audio is truncated to no more than window_size_ms before that line.

**Pull request merged**